node/src/mdns.rs

```diff
@@ -44,6 +43,9 @@ impl NmosMdnsRegistry {
             txt.get("pri"),
         ) {
             // TODO: Validate fields
+            if !["http", "https"].contains(&api_proto.as_str()) {
```

Comment:
There should be a enum expressing HTTP and HTTPS. Why not implement TryFrom for the api_proto type? Seems a bit hacky espcially due to case sensitivity of the comparrison.